### DISCUSSION TITLE
記事を書く時に何をすればいいかわかる記事を用意したい

### DISCUSSION THREAD

**Discussion by neko314** [rubima/magazine.rubyist.net#545] [要望・アイデア]
@shioimm から口頭で、るびまに記事を書きたい人や書くことになった人が、まず最初に何からやるか、どういうふうに記事を作ると良いかわかる記事があると、敷居が下がって良さそう、とご提案いただきました。
確かに！と思ったので考えていきたい。

**Comment by neko314** [comment]
https://github.com/rubima/magazine.rubyist.net/blob/master/doc/writing_process.md の内容を含んだ、もっと具体的な手順があると良いのかなと考えています。

例えば、まずはこのリポジトリに来る、とか。
これまで色々な方とお話ししてみた体感では、るびま自体を最近知った方で「記事書くといいんだなー」って思ってくださった方がいても、このリポジトリに結構気づいていない印象を抱いています。
るびま上に`記事へのご意見・ご感想やるびまへのご要望などありましたら、 [るびま編集部](https://github.com/rubima/magazine.rubyist.net/discussions) までお気軽にご連絡ください。`でリンクはしてありますが、もっと前面に出した記事があったら気づいてもらいやすくなりそうかなと思います。

それ以外にも示しておくと良い情報はある気がします。

**Comment by shokola** [comment]
私も前回の記事書いた時に手順がまとまっているものが欲しかった経験があるので、3、4か月スパンの気長な感じで良ければ担当しますよ〜〜〜😉

**Comment by neko314** [comment]
@shokola 
えっすごいめちゃくちゃありがたいです！！ぜひお願いしたいです🙏🙏

**Comment by shokola** [comment]
ガッテンです！
記事作成前でこの流れで良いですか？漏れないですか？って簡易一覧をここで先に確認するのでお待ちを〜〜。

**Comment by shokola** [comment]
だいたいこんな感じでしょうか？ `writing_process.md` 等の内容は随時変わっていくと思いますので、　内容を直接書かずにリンクを貼るのがいいかなと思っています🙇‍♀️



# **るびまを書いてみよう**と思ったあなたへ


## 担当コンテンツの決定

### ネタはないが書いてみたい

[Discussions](https://github.com/rubima/magazine.rubyist.net/discussions) をのぞいて担当が決まってないものに挙手コメントをしよう

### 書きたいネタがある

[Discussions/記事を書きたい](https://github.com/rubima/magazine.rubyist.net/discussions/categories/%E8%A8%98%E4%BA%8B%E3%82%92%E6%9B%B8%E3%81%8D%E3%81%9F%E3%81%84) にて新規discussionを作成して採用されるか確認しよう


## 執筆作業

以下のリンクを参考に記事を作成、Pull Requestを作成し、レビューに出します

https://github.com/rubima/magazine.rubyist.net/blob/master/doc/writing_process.md

### 注意点

書く前にポリシーを読んで全体的な文章の統一を意識してください
https://github.com/rubima/magazine.rubyist.net/blob/master/doc/editing_policy.md

**Comment by neko314** [comment]
@shokola 
大変素晴らしい...!ありがとうございます！

`るびまを書いてみようと思ったあなたへ`の中に、前段としてるびまはGitHubのリポジトリがありますよっていうこと自体も書いておきたいです！
今まで何名かとやりとりしていて「るびまでもPR作ればいいんですか！（目から鱗）」というような反応をいただくことが度々あり。それ自体は全然何も悪いことではないんですが、もっとリーチする方法を考えたいなって思ったのがこのdiscossionを立てた経緯となりました。

[サイト](https://magazine.rubyist.net/)のFAQやフッターにはリポジトリへのリンクが貼ってあるのですが、もうちょっと目立たせたり、記事として書いておくことで「ここを読んで！」っていつでも誰でも言えるようにしたいなとイメージしてます ☺️

**Reply by shokola** [reply]
なるほど！OSSですってことをまず伝えたほうがいいですね！OSSだとわかれば皆Welcomeだってことも自然と伝わりますしね。

それも入れて本文作りますね〜🖐️🖐️

**Reply by neko314** [reply]
ありがとうございます！よろしくお願いします 😊

**Comment by neko314** [comment]
#601 にそってissueを立てました。https://github.com/rubima/magazine.rubyist.net/issues/603 やり取りを確認するにはissueからこのdiscussionを辿る必要があるのですが🙇
こちらはcloseしますね。
